composer/callbacks/checkpoint_saver.py

```diff
@@ -321,6 +328,31 @@ def __init__(
 
         self.start_batch = None
 
+        self.remote_uploader = None
+        backend, _, _ = parse_uri(save_folder)
+        self.remote_uploader_futures: List[List[Future]] = []
+        self.symlink_file_tasks: List[tuple[str, str]] = []
+        self.this_rank_saves_remote_symlinks: bool = False
+        self.tmp_dir_for_symlink = tempfile.TemporaryDirectory()
+        self.num_concurrent_uploads = num_concurrent_uploads
+
+        if backend != '':
+            if backend == 'wandb':
+                raise NotImplementedError(
+                    f'There is no implementation for WandB via URI. Please use '
+                    'WandBLogger with log_artifacts set to True',
```

Comment:
doesn't this not work since you don't call `upload_file`?

Reply:
I just make this same as existing RUD, which doesn't support wandb https://github.com/mosaicml/composer/blob/13f2a4fab92ef33f4c490a31f17c608ddb1a7eee/composer/utils/file_helpers.py#L478-L482